### ISSUE TITLE
Teach extra DOM APIs

### ISSUE THREAD

**Issue by oliverjam** [foundersandcoders/DOM-manipulation-Challenge#10]
A few common and useful DOM methods/properties are not covered in the README.

### Class names
We cover how to access `element.className`, but we don't really make it clear that this property is always a single string, even for many class names. It's difficult to add multiple class names/toggle them without knowing about the `element.classList` API.

### Text content
It's also very common to read and write `element.textContent`. One of the exercises requires reading this, so maybe we should have an example of it? Or are we expecting 	students to click through to the MDN links and browse around?

### Creating elements
The final exercise asks students to create a `ul` element. Some of them will probably never have seen `document.createElement` before, so maybe we should document this briefly in the README, rather than hoping they have time to google around.

**Comment by zurda** [comment]
This is great @oliverjam ! I feel this is exactly what's missing from this mc. 

Since it does make the it a bit longer for a morning challenge, I was thinking about creating an `index.html` file with all the README examples commented out. This way even if we can't go through the entire solution in class - we could open a browser, comment in the lines, and quickly show them everything in the console. I don't know if this makes sense as a mentor notes branch, but I'm happy to create (and I think it'll help me while mentoring this week).

**Comment by oliverjam** [comment]
Yeah I think maybe an `examples.html` would be a good idea. That way the README can be focused on explaining the bare minimum needed to start the exercise, and if students need a bit more help they can check the examples.

If you want to create that I'm happy to review a PR. Feel free to copy examples from my DOM guide if it helps
